cms/Cargo.toml

```
@@ -16,17 +16,27 @@ rust-version = "1.65"
 [dependencies]
 der = { version = "0.7.6", features = ["alloc", "derive", "oid", "pem"] }
 spki = { version = "0.7" }
-x509-cert = { version = "0.2", default-features = false, features = ["pem"] }
+x509-cert = { version = "0.2.2-pre", default-features = false, features = ["pem"], path = "../x509-cert"}
```

```suggestion
x509-cert = { version = "0.2.3", default-features = false, features = ["pem"], path = "../x509-cert"}
```

Fresh off the press!